nodeapp/docker-compose.yml

```diff
@@ -25,7 +26,7 @@ services:
 
   tor:
     build: ../docker/tor
-    container_name: tor-dev
+    container_name: tor-dev-frontend
```

Comment:
Changing names allows to have a docker group for `fullstack` and another docker group for `frontend`